subsys/bluetooth/host/hci_core.c

```diff
@@ -552,13 +552,17 @@ int bt_le_create_conn_ext(const struct bt_conn *conn)
 	} else {
 		const bt_addr_le_t *peer_addr = &conn->le.dst;
 
-#if defined(CONFIG_BT_SMP)
-		if (!bt_dev.le.rl_size ||
-		    bt_dev.le.rl_entries > bt_dev.le.rl_size) {
+		if (bt_addr_le_cmp(&conn->le.resp_addr, BT_ADDR_LE_ANY)) {
+			char resp_addr_str[BT_ADDR_LE_STR_LEN];
```

Comment:
This is duplicated, you could create a static fn, e.g., `static void log_addr(char* message, bt_addr_le_t addr)` instead.